spec/Section 3 -- Type System.md

```diff
@@ -1659,6 +1681,21 @@ is constructed with the following rules:
   does not provide a default value, the input object field definition's default
   value should be used.
 
+Further, if the input object is a OneOf Input Object, the following additional
+rules apply:
+
+- If the input object literal or unordered map does not contain exactly one
+  entry an error must be thrown.
+
+- Within the input object literal or unordered map, if the single entry is
+  {null} an error must be thrown.
+
+- If the coerced unordered map does not contain exactly one entry an error must
+  be thrown.
+
+- If the value of the single entry in the coerced unordered map is {null} an
+  error must be thrown.
```

Comment:
What's the difference between the first two conditions and the second two conditions?

Reply:
The first two apply before coercion, the second two apply after coercion.

Reply:
See https://github.com/graphql/graphql-spec/pull/825#discussion_r588879692 discussion there still seems somewhat open, pending input from @leebyron